http_service/tests/conftest.py

```diff
@@ -230,4 +234,69 @@ def mock_repo(tmpdir, monkeypatch):
         repo.add([str(test_file).encode("utf-8")])
         repo.commit(f"Base history {i}", user="bugbug")
 
+    # Allow using the local code analysis server.
+    responses.add_passthru("http://127.0.0.1")
+
     return repo_dir, repo
+
+
+@pytest.fixture
+def mock_component_taskcluster_artifact():
+    responses.add(
+        responses.HEAD,
+        "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.mozilla-central.latest.source.source-bugzilla-info/artifacts/public/components.json",
+        status=200,
+        headers={"ETag": "100"},
+    )
+
+    responses.add(
+        responses.GET,
+        "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.mozilla-central.latest.source.source-bugzilla-info/artifacts/public/components.json",
+        status=200,
+        json={},
+    )
+
+
+@pytest.fixture
+def mock_schedule_tests_classify(tmpdir, monkeypatch):
```

Comment:
This fixture looks like it would be better off in `test_schedule_tests.py`

Reply:
Yeah, there are a couple more that might make sense to move in test_schedule_tests.py, we can do it in a follow-up as I don't want to cause too much conflicts with @La0's PR.